packages/keystone/databaseAdapters/BalancingReplicaKnexAdapter/adapter.js

```
@@ -13,179 +13,260 @@
-                // NOTE: builder.toSQL() in single-query case will return object of shape:
-                // { method: "<knex-method>", sql: "select * from ... limit ?", bindings: [100] }
-                // parser cannot understand bindings, so we need to do some tricks, which Client_PG does under the hood
-                const sqlQueryWithPositionalBindings = this.knex.client.positionBindings(sqlObject.sql)
-
-                const selectedPool = this._selectTargetPool(sqlQueryWithPositionalBindings)
-
-                return selectedPool.getQueryRunner(builder)
-            } catch (err) {
-                logger.error({ msg: 'Unexpected error happened during SQL query routing', err })
-                throw new Error(`Unexpected error happened during SQL query routing: ${String(err)}`)
+          try {
```

ESLint

[error] Expected indentation of 12 spaces but found 10.

(@typescript-eslint/indent)